feedback-form/README.md

```diff
@@ -30,47 +39,103 @@ class Feedback
 
   validates :email, :name, :message, presence: true
   validates :email, format: Devise.email_regexp
-
-  def save
-    valid?
-  end
 end
 ```
 
 ```ruby
-# app/views/feedbacks/new.html.erb
-- title("Contact Us")
-- description("Contact the team")
-- keywords("Contact, Email, Touch, Getting in touch")
-- meta robots: "noindex"
+# app/decorators/feedback_decorator.rb
+class FeedbackDecorator < ApplicationDecorator
+  delegate :user, :phone, :message
 
-.row
-  .columns
-    h2 Feedback
+  def author
+    user ? user.full_name : ""
+  end
 
-.row
-  .medium-6.columns
-    = simple_form_for(feedback, url: feedback_path) do |f|
+  def email
+    user ? user.email : ""
+  end
+end
+```
 
-      legend
-        ' Have a question?
-        | You may find the answer in the #{link_to("FAQ", "#")}.
+```ruby
+# app/interactors/hipchat_interactor/organizer.rb
+module HipchatInteractor
+  class Organizer
+    include Interactor::Organizer
 
-      .form-inputs
-        = f.input :name
-        = f.input :email
-        = f.input :phone
-        = f.input :message, as: :text
+    organize HipchatInteractor::Connector, HipchatInteractor::Messenger
+  end
+end
+```
 
-      .form-actions
-        = f.button :submit, "Submit"
+```ruby
+# app/interactors/hipchat_interactor/connector.rb
+module HipchatInteractor
```

Comment:
I don't see any reason to use `Connector` as Interactor. Intercators should incapsulate business logic.
Better create separate class to wrap `HipChat::Client`